src/app/views/profile/overview/overview.vue

```diff
@@ -375,21 +375,20 @@
 
 .-communities
 	display: grid
-	grid-template-columns: repeat(5, 55px)
-	grid-gap: 5px 10px
+	grid-template-columns: repeat(5, minmax(55px, 1fr))
+	grid-gap: 10px
```

Comment:
let's switch to a 8px gap, or a 12px one

Reply:
switched to 8px